Makefile

```diff
@@ -99,7 +99,7 @@ fix-style-webapp:
 	
 test-server: vendor
 	echo Running server tests
-	go test -v -coverprofile=coverage.txt ./...
+	go test -gcflags=-l -v -coverprofile=coverage.txt ./...
```

Comment:
What was the need for this flag?

Reply:
this was included to fix the failing test cases which were using the monkey patch